pkg/controller/issueapi/issue.go

```diff
@@ -68,11 +68,26 @@ func (ic *IssueAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Verify SMS configuration
+	smsProvider, err := ic.db.GetSMSProvider("") // TODO(sethvargo): pass in realm
```

Comment:
would be good to put a write through cache in front of these (can be future PR)

Reply:
Done